### ISSUE TITLE
Non-descriptive link text with non-translateable aria label.

### ISSUE THREAD

**Issue by bisd** [localgovdrupal/localgov_base#490]
This issue occurs on the following page:

https://localhost:32772/births-deaths-marriages-and-citizenship/status

The link text "Read more" is not descriptive. Although there is an associated unique aria label there is no option to translate. Since screen-readers do not read English in a phonetically correct way, in a different language it would pronounce the English a different way which makes it unlikely to pass audit. 

Adding an option to translate through GETTEXT function [_e( 'ARIA-LABEL', 'name_of_theme' )] to the aria-labels would allow web developers to translate the strings in aria-labels into other languages and improve the chances of scoring better at accessibility audits.

The aria label is generated in the "/block/service-status-page.html.twig" template.

@msayoung 

![aria-label](https://github.com/localgovdrupal/localgov_base/assets/2633608/47bb7f7c-3809-425f-bf84-524e82160a55)

**Comment by ctorgalson** [comment]
Hi @bisd,

Judging by the screenshot (which [matches the current codebase](https://github.com/localgovdrupal/localgov_base/blob/1.x/templates/block/service-status-block.html.twig#L35)), the `aria-label` attribute's value and the link's text should both be translatable (it's the `|t` that indicates that).

Drupal does have to 'learn' that there are template values to be translated though. See, for example, this recent Stack Exchange post: https://drupal.stackexchange.com/questions/192498/how-do-i-translate-strings-in-twig-templates

**Comment by bisd** [comment]
@ctorgalson many thanks for confirming. I guess it just needs the content of the "Read more" link changed then?

**Comment by markconroy** [comment]
The content of the "Read more" link should be translatable. 

Here's the code:

```
<a class="service-status__read-more" aria-label="{{ 'Read more about status:'|t }} {{ item.title }}" href="{{ item.url }}">{{ 'Read more'|t }}</a>

```

I'm going to close this issue. Feel free to re-open if needed.

---
Thanks to [Big Blue Door](https://www.bigbluedoor.net/) for sponsoring my time to work on this.
